src/QueryBuilder/AttributesQueryBuilder/AttributesTypeNumberQueryBuilder.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusElasticsearchPlugin\QueryBuilder\AttributesQueryBuilder;
+
+use Elastica\Query\BoolQuery;
+use Elastica\Query\Term;
+
+class AttributesTypeNumberQueryBuilder implements AttributesQueryBuilderCollectorInterface
```

Comment:
I would rename all classes inside using a rule as below:

```suggestion
class NumberAttributesQueryBuilder implements AttributesQueryBuilderCollectorInterface
```

Reply:
It still pops up